src/app/onboarding/components/create-archive-for-me-screen/create-archive-for-me-screen.component.ts

```diff
@@ -0,0 +1,32 @@
+/* @format */
+import { Component, Output, EventEmitter } from '@angular/core';
+import { AccountService } from '@shared/services/account/account.service';
+
+@Component({
+  selector: 'pr-create-archive-for-me-screen',
+  templateUrl: './create-archive-for-me-screen.component.html',
+  styleUrl: './create-archive-for-me-screen.component.scss',
+})
+export class CreateArchiveForMeScreenComponent {
+  private readonly TYPE = 'type.archive.person';
+  public name = '';
+
+  @Output() goBackOutput = new EventEmitter<string>();
+  @Output() continueOutput = new EventEmitter<Record<string, string>>();
```

Comment:
Can the typing on this be a bit more specific instead of `Record<string, string>`? This is a pretty specific type so I'm not sure what to call it (`ArchiveCreatedEvent`? I don't know!) but I'd like something a bit more locked down than this.